web/.eslintrc.json

```diff
@@ -144,7 +144,9 @@
     ],
     "unicorn/no-nested-ternary": ["error"],
     "unicorn/no-useless-undefined": "off",
-    "unicorn/no-null": "off"
+    "unicorn/no-null": "off",
+    // typescript doesn't recognize at syntax yet
+    "unicorn/prefer-at": 0
```

Comment:
This was added to TypeScript 4.6 and we are already using it in several places.
See https://devblogs.microsoft.com/typescript/announcing-typescript-4-6/#target-es2022